client/components/tinymce/plugins/wpcom-view/views/simple-payments/index.jsx

```diff
@@ -26,83 +26,76 @@ import { FEATURE_SIMPLE_PAYMENTS } from 'lib/plans/constants';
 
 class SimplePaymentsView extends Component {
 	render() {
-		const {
-			translate,
-			productId,
-			product,
-			siteId,
-			sitePlan,
-			planHasSimplePaymentsFeature,
-		} = this.props;
-
-		if ( ! product ) {
-			return <QuerySimplePayments siteId={ siteId } productId={ productId } />;
-		}
+		const { productId, product, siteId } = this.props;
+
+		return (
+			<div className="wpview-content wpview-type-simple-payments">
+				<QuerySimplePayments siteId={ siteId } productId={ productId } />
```

Comment:
Should we check if `productId` is available or update `QuerySimplePayments` to have an `all` param? Currently if `productId` is undefined, it'll request all products.